### PR TITLE
fixed the login bug

### LINKED ISSUE #1: Login is broken.
Looks like we are using the wrong route

### PR DESCRIPTION
**Pull request by nadrane**
closes #1

### CODE REVIEW

**Review by halcarleton — CHANGES_REQUESTED** [review]
You should probably display the username Nick

File: app/components/WhoAmI.jsx [hunk 1]
```diff
@@ -2,7 +2,7 @@ import React from 'react'
 
 export const WhoAmI = ({ user, logout }) => (
   <div className="whoami">
-    <span className="whoami-user-name">{user && user.name}</span>
+    <span className="whoami-user-name">username</span>
```
Comment (nadrane) [inline app/components/WhoAmI.jsx]: Nick, this looks wrong

**Comment by nadrane** [comment]
Hal, I fixed it. take a look

**Review by halcarleton — APPROVED** [review]
Great Job!

**Pull request merged**